lib/MusicBrainz/Server/Entity/Editor.pm

```diff
@@ -264,6 +264,7 @@ around TO_JSON => sub {
 
     return {
         %{$self->$orig},
+        email => $self->email,
```

Comment:
Something to be wary about: we'll eventually want a sanitized version of these TO_JSON methods so the output can be embedded into a page or outputted in /ws/js/edit.